spec/requests/api/v2/providers_spec.rb

```diff
@@ -106,5 +106,18 @@
         )
       end
     end
+
+    context 'with unalphabetical ordering in the database' do
```

Comment:
Spec can just be `context '. in_order' do`

Reply:
It's a request/integration spec though, shouldn't it be more high-level in its verbiage, present the specs in terms closer to an actual user journey?

Reply:
Sorted, have added a model spec for the scope 👍